### PR TITLE
Added Ciena Waveserver

### LINKED ISSUE #8183: Add Support - Ciena Waveserver
DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for known/validated bugs, please do not post issues asking for help or how to do X, Y or Z. 

You can use our [Discord server](https://t.libren.ms/discord) to ask questions or our [community site](https://community.librenms.org).

If you have a feature request please post this on our [community site](https://community.librenms.org/c/feature-requests).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [ ] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [ ] Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
- [ ] Please provide ALL info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [ ] Please provide as much detail as possible.
- [ ] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Love librenms? Please consider supporting our collective:
👉  https://t.libren.ms/donations

====================================
Component | Version
--------- | -------
LibreNMS  | 1.36-80-ga6a1f08
DB Schema | 232
PHP       | 7.0.25
MySQL     | 5.5.56-MariaDB
RRDTool   | 1.4.8
SNMP      | NET-SNMP 5.7.2
====================================

Please add support for the Ciena Waveserver device.

https://p.libren.ms/view/d21927f7 (Discovery)
https://p.libren.ms/view/1fee7a29 (Poller)
https://p.libren.ms/view/1bf4f9ca (SNMP Walk)

### PR DESCRIPTION
**Pull request by PipoCanaja**
fixes #8183

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/ciena-waveserver.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                -
+                    oid: cwsXcvrTemperatureTable
+                    value: cwsXcvrTemperatureActual
+                    num_oid: '.1.3.6.1.4.1.1271.3.4.15.13.1.2.{{ $index }}'
+                    descr: 'Module {{ $subindex0 }} - {{ $cwsXcvrCienaIdDescription }}'
+                    index: 'cwsXcvrTemperatureTable.{{ $index }}'
+                    group: 'Modules'
+                    skip_values:
+                        -
+                            oid: cwsXcvrTemperatureThresholdHighAlarmThreshold
```
Comment (murrant) [inline includes/definitions/discovery/ciena-waveserver.yaml]: why not set the threshold(s)?

File: includes/definitions/discovery/ciena-waveserver.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+mib: CIENA-WS-XCVR-MIB
+modules:
+    sensors:
+        pre-cache:
+            data:
+                - oid:
+                    - cwsXcvrTemperatureThresholdHighAlarmThreshold
+                    - cwsXcvrCienaIdDescription
+                    - cwsXcvrCienaIdCienaItemNumber
```
Comment (murrant) [inline includes/definitions/discovery/ciena-waveserver.yaml]: cwsXcvrCienaIdCienaItemNumber seems unused

Comment (PipoCanaja) [reply includes/definitions/discovery/ciena-waveserver.yaml]: yep. That's a forgotten line ;)

File: includes/definitions/discovery/ciena-waveserver.yaml (hunk 1 above)
Comment (PipoCanaja) [reply includes/definitions/discovery/ciena-waveserver.yaml]: done

**Comment by murrant** [comment]
@PipoCanaja if you specify some text like this in your PR description, it will auto-close the issue `fixes #8183`

**Comment by PipoCanaja** [comment]
> @PipoCanaja if you specify some text like this in your PR description, it will auto-close the issue `fixes #8183`

OK !

**Pull request merged**